src/SdkCommon/ClientRuntime.Azure/Tests/CR.Azure.NetCore.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using Xunit;
+
+[assembly: AssemblyTitle("Microsoft Rest Azure Client Runtime Tests")]
+[assembly: AssemblyDescription("Tests for the Azure Client Runtime.")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.4.2.0")]
```

Comment:
@dsgouda is this typo that the assemblyFileVersion is different here?
We don't even want this version to be changed.